modules/core/core-api/src/main/java/com/enonic/xp/content/Content.java

```diff
@@ -444,31 +444,31 @@ protected Builder()
 
         protected Builder( final Content source )
         {
-            this.id = source.id;
-            this.valid = source.valid;
-            this.parentPath = source.parentPath;
-            this.name = source.name;
-            this.type = source.type;
-            this.data = source.data != null ? source.data.copy() : null;
-            this.attachments = source.attachments;
-            this.extraDatas = source.extraDatas != null ? source.extraDatas.copy() : null;
-            this.displayName = source.displayName;
-            this.owner = source.owner;
-            this.createdTime = source.createdTime;
-            this.modifiedTime = source.modifiedTime;
-            this.creator = source.creator;
-            this.modifier = source.modifier;
-            this.hasChildren = source.hasChildren;
-            this.page = source.page != null ? source.page.copy() : null;
-            this.thumbnail = source.thumbnail;
-            this.childOrder = source.childOrder;
-            this.permissions = source.permissions;
-            this.inheritPermissions = source.inheritPermissions;
-            this.language = source.language;
-            this.contentState = source.contentState;
-            this.publishInfo = source.publishInfo;
-            this.processedReferences = ContentIds.create().addAll( source.processedReferences );
-            this.workflowInfo = source.workflowInfo;
+            this.id = source.getId();
```

Comment:
getters to class of itself look very suspicions. What was wrong with direct field access?